src/config.js

```diff
@@ -0,0 +1,17 @@
+/**
+* @fileoverview Mixpanel source code, modified for DSP requirements
+*
+* Derived from code in https://github.com/mixpanel/mixpanel-js
+*/
+
+var Config = {
+  DEBUG: false,
+  LIB_VERSION: '0.0.1',
```

Comment:
Is this intended to match the version in `package.json`?

Reply:
Yes, done in 20edf62.  

In another library I maintain, I [populate version strings in app code from `package.json`](https://github.com/eweitz/ideogram/blob/master/scripts/js/prepublish.js) as part of a `prepublish` NPM script.  I might incorporate that here later, but for now aligning these two copies is manual.